frontend/src/locales/fr.po

```diff
@@ -273,23 +273,23 @@ msgstr "Archivé"
 
 #: src/dialog/photo/archive.vue:10
 msgid "Are you sure you want to archive the selection?"
-msgstr "Êtes-vous sûr de vouloir archiver la sélection?"
+msgstr "Êtes-vous sûr de vouloir archiver la sélection ?"
 
 #: src/dialog/album/delete.vue:10
 msgid "Are you sure you want to delete these albums?"
-msgstr "Êtes-vous sûr de vouloir supprimer ces albums?"
+msgstr "Êtes-vous sûr de vouloir supprimer ces albums ?"
 
 #: src/dialog/label/delete.vue:10
 msgid "Are you sure you want to delete these labels?"
-msgstr "Êtes-vous sûr de vouloir supprimer ces étiquettes?"
+msgstr "Êtes-vous sûr de vouloir supprimer ces étiquettes ?"
 
 #: src/dialog/account/remove.vue:10
 msgid "Are you sure you want to delete this account?"
-msgstr "Êtes-vous sûr de vouloir supprimer ce compte?"
+msgstr "Êtes-vous sûr de vouloir supprimer ce compte ?"
 
 #: src/dialog/file/delete.vue:10
 msgid "Are you sure you want to permanently delete this file?"
-msgstr "Êtes-vous sûr de vouloir supprimer définitivement ce fichier?"
+msgstr "Êtes-vous sûr de vouloir supprimer définitivement ce fichier ?"
```

Comment:
Why the blank space before ?

Reply:
I was curious about this, so I googled it: https://iwillteachyoualanguage.com/learn/french/french-tips/french-punctuation

Reply:
Yes, in French there is a space before question or exclamation mark. Actually a non-breaking space should be used.

Reply:
Interesting! Some Germans do it too, but it's wrong over here... there's even a Wikipedia article on this:

https://en.wikipedia.org/wiki/Plenken